src/platform/datapath_winuser.c

```diff
@@ -3126,6 +3167,10 @@ CxPlatDataPathUdpRecvComplete(
                 } else if (CMsg->cmsg_type == IPV6_ECN) {
                     ECN = *(PINT)WSA_CMSG_DATA(CMsg);
                     CXPLAT_DBG_ASSERT(ECN < UINT8_MAX);
+                } else if (CMsg->cmsg_type == IPV6_HOPLIMIT) {
+                    HopLimitTTL = *(PINT)WSA_CMSG_DATA(CMsg);
+                    CXPLAT_DBG_ASSERT(HopLimitTTL <= 256);
+                    // printf("HopLimitTTL: %d \n", HopLimitTTL);
```

Comment:
I will remove these once all platforms are supported. There is still a pending TODO in `Connection.c`